gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dlabel_ttt2.lua

```diff
@@ -92,6 +92,26 @@ function PANEL:Paint(w, h)
 	return true
 end
 
+---
+-- @param Panel slave
+-- @realm client
+function PANEL:SetMaster(master)
+	if not IsValid(master) then return end
+
+	self.master = master
+end
+
+---
+-- @return number
+-- @realm client
+function PANEL:GetMargin()
```

Comment:
Usually margin means top, left, right, bottom.

So I'd suggest renaming this to clarify that this is an indentation margin